apps/web/src/index.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import './index.css';
+// import './index.css'; // to be removed?
```

Comment:
To be removed?

Reply:
not sure haha, that was in the web2 already